File: src/tests/Markify.Fixtures/Projects/Source/Struct/StructProperties.vb
Public Structure FooType
    Public Property AutoProperty()

    Public ReadOnly Property ReadOnlyProperty() As Integer

    Public WriteOnly Property WriteOnlyProperty() As Integer
        Set
        End Set
    End Property

    Public Property WithGetterModifierProperty() As Integer
        Friend Get
            Return 1
        End Get
        Set(value As Integer)
        End Set
    End Property

    Public Property WithSetterModifierProperty() As Integer
        Get
            Return 1
        End Get
        Friend Set(value As Integer)
        End Set
    End Property

    Public Shared ReadOnly Property WithInitialValueProperty() As Integer = 1

    Protected Property WithModifierProperty() As Integer

    Property WithNoModifierProperty() As Integer

    Public Shared Property StaticProperty() As Integer
End Structure

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Struct/StructProperties.cs
public struct FooType
{
    public int AutoProperty { get; set; }

    public int ReadOnlyProperty { get; }

    public int WithExpressionBody => 1;

    public int WriteOnlyProperty
    {
        set { }
    }

    public int WithGetterModifierProperty
    {
        internal get { return 1; }
        set { }
    }

    public int WithSetterModifierProperty
    {
        get { return 1; }
        internal set { }
    }

    public static int WithInitialValueProperty { get; } = 1;

    protected int WithModifierProperty { get; set; }

    int WithNoModifierProperty { get; set; }

    public static int StaticProperty { get; set; }
}